src/main/java/net/dv8tion/jda/api/entities/Message.java

```diff
@@ -1179,6 +1181,311 @@ default boolean isFromGuild()
+     *             <li>If the provided {@link net.dv8tion.jda.api.entities.Emote Emote} is fake {@link net.dv8tion.jda.api.entities.Emote#isFake() Emote.isFake()}.</li>
+     *             <li>If the provided {@link net.dv8tion.jda.api.entities.Emote Emote} cannot be used in the current channel.
+     *                 See {@link Emote#canInteract(User, MessageChannel)} or {@link Emote#canInteract(Member)} for more information.</li>
+     *         </ul>
+     *
+     *  @return The {@link net.dv8tion.jda.api.requests.restaction.pagination.ReactionPaginationAction ReactionPaginationAction} of the emote's users.
+     */
+    @Nonnull
+    @CheckReturnValue
+    ReactionPaginationAction retrieveReactionUsers(Emote emote);
+
+    /**
+     * This obtains the {@link net.dv8tion.jda.api.entities.User users} who reacted using the given unicode emote.
+     *
+     * <p>Messages keep track of reactions as well as who added those reactions. Using this data, we can obtain a
+     * {@link net.dv8tion.jda.api.requests.restaction.pagination.ReactionPaginationAction ReactionPaginationAction}
+     * of the reacting users.</p>
+     *
+     * @param unicode
+     *      The unicode emote to retrieve users for.
+     *
+     * @throws java.lang.IllegalArgumentException
+     *      If the provided unicode emote is null or empty.
+     *
+     *  @return The {@link net.dv8tion.jda.api.requests.restaction.pagination.ReactionPaginationAction ReactionPaginationAction} of the emote's users.
+     */
+    @Nonnull
+    @CheckReturnValue
+    ReactionPaginationAction retrieveReactionUsers(String unicode);
+
+    /**
+     *  This obtains the {@link net.dv8tion.jda.api.entities.MessageReaction.ReactionEmote ReactionEmote} for the given reaction name on this message.
+     *
+     * <p>Messages can host a number of different reactions at once. In order to distinguish between reactions they are
+     * represented by their id or name. This method is specifically for referencing unicode emojis that follow the form ":name:"</p>
+     *
+     * @param name
+     *      The name of the reaction emoji.
+     *
+     * @throws java.lang.IllegalArgumentException
+     *      If the provided emote name is null or empty.
+     *
+     * @return The {@link net.dv8tion.jda.api.entities.MessageReaction.ReactionEmote ReactionEmote} of this message.
+     */
+    @Nullable
+    @CheckReturnValue
+    MessageReaction.ReactionEmote getReactionByName(String name);
+
+    /**
+     *  This obtains the {@link net.dv8tion.jda.api.entities.MessageReaction.ReactionEmote ReactionEmote} for the given reaction id on this message.
+     *
+     * <p>Messages can host a number of different reactions at once. In order to distinguish between reactions they are
+     * represented by their id or name. This method is specifically for referencing {@link net.dv8tion.jda.api.entities.Emote custom emotes}
+     * that follow the form "<:name:id>"</p>
```

Comment:
I don't understand what this is trying to tell me, the `<:name:id>` has nothing to do with this method or emotes specifically.

Reply:
Yeah good point, I cleaned up the section a bit